robot/Cumulus/resources/NPSP.py

```diff
@@ -1350,13 +1350,30 @@ def setupdata(self, name, contact_data=None, opportunity_data=None, account_data
             if payment_data is not None:
                 numdays = 30
                 i = 1
+               
                 while i <= int(payment_data['NumPayments']):
                     payment_schedule_data = {}
                     numdays = numdays*2
-                    scheduled_date =  (datetime.now() + timedelta(days = numdays)).strftime('%Y-%m-%d')
-                    payment_schedule_data.update( {'npe01__Opportunity__c' : data[f"{name}_opportunity"]["Id"] , 'npe01__Scheduled_Date__c' : scheduled_date, 'npe01__Payment_Amount__c' : payment_data['Amount'] } )
+                    
```

Comment:
can u add a comment u explain what is being done here

Reply:
Add required comments